tests/test_data_languages.py

```diff
@@ -336,3 +336,12 @@ def test_language_name_structure():
         pytest.fail(
             f'Languages names without expected structure ("LANGUAGE, MODIFIER (SCRIPT)"): {misstructured_language_names}'
         )
+
+
+@pytest.mark.parametrize("lang_code", LANGUAGES)
+def test_id_well_formed(lang_code):
+    if lang_code in ["tw_akuapem_Latn"]:
+        pytest.xfail("Well we need to have a conversation about that")
```

Comment:
☝️

Reply:
Yeah. We don't really have a schema for *dialects* in the same way that we have a schema for regional variants. I need understand from @moyogo why we want Akuapem Twi to be this and not, say, just plain `tw_Latn` (I think it's the prestige dialect) - particularly since we don't have any other `tw_Latn`s to distinguish this from.